src/CppParser/Parser.cpp

```diff
@@ -63,12 +64,23 @@
 using namespace CppSharp::CppParser;
 
 // We use this as a placeholder for pointer values that should be ignored.
-void* IgnorePtr = (void*) 0x1;
+void* IgnorePtr = reinterpret_cast<void*>(0x1);
+
+static bool initializedLLVM = false;
 
 //-----------------------------------//
 
 Parser::Parser(CppParserOptions* Opts) : lib(Opts->ASTContext), opts(Opts), index(0)
 {
+	if (!initializedLLVM)
+	{
+		llvm::InitializeAllTargets();
+		llvm::InitializeAllTargetMCs();
+		llvm::InitializeAllAsmPrinters();
+		llvm::InitializeAllAsmParsers();
+
+		initialized_llvm = true;
```

Comment:
Please build locally before updating your pull requests. You renamed the field (and manually at that rather than with the refactoring tool of the IDE) but you forgot this one usage. The result is https://ci.appveyor.com/project/tritao/cppsharp/build/1.0.2631#L102 .